vectoriadb-index/src/main/java/jetbrains/vectoriadb/index/Quantizer.java

```diff
@@ -30,10 +30,10 @@ public interface Quantizer extends AutoCloseable {
     float[] decodeVector(byte[] vectors, int vectorIndex);
 
     IntArrayList[] splitVectorsByPartitions(int numClusters, int iterations, DistanceFunction distanceFunction,
-                                            @NotNull ProgressTracker progressTracker);
+                                            boolean initializeCentroidsWithKMeansPlusPlus, @NotNull ProgressTracker progressTracker);
```

Comment:
This `initializeCentroidsWithKMeansPlusPlus` param is temporary, just for the benchmark to assess the difference between the random and k-means++ initialisation methods.

When we decide that k-means++ is good enough, I will remove these params and leave only the k-means++ initialisation implementation.

I do not see much sense in keeping both implementations. But if you think we should keep both of them, I will refactor the API more carefully.